src/styles/views/Game.scss

```diff
@@ -9,51 +9,57 @@
 
 .game {
   &.container {
-    background: black;
-    padding: 1.5em;
+    margin: 1em;
     border-radius: $borderRadius;
     display: flex;
     flex-direction: column;
     align-items: center;
+    background-color: $background;
     box-shadow: $dropShadow;
   }
-  &.user-list {
-    list-style: none;
-    padding-left: 0;
-  }
-  &.user-item {
+}
+
+.wordmergebar {
+  &.container {
+    border-radius: $borderRadius;
     display: flex;
-    flex-direction: column;
+    flex-direction: row;
     align-items: center;
-    justify-content: center;
+    background-color: #454545;
+    margin: 0.5em;
+    box-shadow: $dropShadow;
   }
-  & p,h2 {
-    margin: 0.5em 0;
+
+  &.symbol {
+    font-size: 5em;
+    text-align: center;
+    display: flex;
   }
 }
 
-.player {
+.wordboard {
   &.container {
```

Comment:
The current implementation doesn't handle either horizontal or vertical overflow. Horizontal we may have to discuss because we can restrict the max length of a word in the backend, making this more predictable. As for vertical overflow, you may use this CSS:

```js
.wordboard {
  &.container {
    padding: 0.5em;
    border-radius: $borderRadius;
    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: #101010;
    margin: 0.5em;
    box-shadow: $dropShadow;
    overflow-y: auto;  //Will automatically had a scrollbar if the max-height was exceeded, prevents page overflow. 
    overflow-x: hidden;
    max-height: 300px; 
  }
}

```